### DISCUSSION TITLE
Pertanyaan terkait web dashboard

### DISCUSSION THREAD

**Discussion by Samgoesniady** [teamalgoritma/community#406] [Q&A] [answered]
Selamat siang kakak" semua ijin bertanya bagaimana membuat dashboard shiny terhubung dengan server dam bisa diupdate datanya secara real time?

**Comment by jafarhf1** [comment, marked as answer]
Dear @Samgoesniady,
Terkait pertanyaan bapak, jawabannya adalah bisa dan berikut adalah referensi yang bisa kami berikan:
https://rviews.rstudio.com/2017/11/15/shiny-and-scheduled-data-r/

Demikian yang bisa kami sampaikan, semoga membantu.

**Reply by Samgoesniady** [reply]
Baik kak terima kasih

**Comment by amaliasf** [comment]
Menjawab pertanyaan Pak @Samgoesniady pada sesi mentoring terkait menemukan data latitude dan longitude pada daerah tertentu, misalkan contohnya daerah Maluku, bisa refer ke link berikut https://gadm.org/download_country_v3.html. Datanya dalam bentuk shape file, jadi perlu di baca dulu dengan R menggunakan function `readRDS()`

Note: Level dari shapefile umumnya berdasarkan batas administratif negara diantaranya level 0 (negara) level 1 (provinsi), level 2 (kota), dst. 

Untuk Maluku, karena merupakan provinsi maka bisa dipilih level 1.

Semoga menjawab dan lancar untuk eksplorasinyaa :)
